README.md

```diff
@@ -50,17 +108,23 @@ Launch development server
 
 ```bash
 php app/console server:run #localhost only
-php app/console server:run 0.0.0.0:9001 #available for everyone 
+php app/console server:run 0.0.0.0 #available for everyone on port 8000
 ```
-(you can also use the php builtin development web server : `cd web && php -S localhost:9001`)
+(you can also use the php builtin development web server : `cd web && php -S localhost:8000`)
+
+
+### Windows
 
-Deploy on shared web hosting services
--------------------------------------
+TODO
+http://www.wampserver.com/ should be able to provide everything you need.
+
+
+## Deploy on shared web hosting services
```

Comment:
I don't know if it makes sense to keep this part. It has been written at a time where our website was hosted on an OVH PHP hosting where we didn't have any access to a shell.
So it was really hard to deploy Symfony on this platform and I put my hacks in these scripts to "ease" the deployment.
Finally, we moved to a VPS for performance and simplicity reasons.
If you don't use them, I am really encouraging you to:
  1. Remove this part of the documentation
  2. Remove the `./deploy` folder

In any case, I must warn you that even 4 years ago these scripts were not reliable to deploy.
And I think that today it can be only worse.

Reply:
Ok, thank you for your  advice. We will remove that part.